CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contribution Guidelines for TCD XIOS Demonstration
+
+For code contributions, please create your own Fork of this repository and propose code via Pull Request, as detailed in [repositories guidance](repositories.md)
+
+All code contributions will require a code review prior to merging to `main`.
+
+Pull Requests should test your code contributions with respect to several versions of XIOS, and provide a report within the PR. All tests must pass for merging to be allowed.
+Please ensure that all new directories have an `__init__.py` to ensure they are 'discovered' by Python's `unittest` and suitable tests of the contributed code.
+
+## Licensing
+By contributing code to this repository, You hereby grant to the Met Office a perpetual, worldwide, non-exclusive, royalty-free, irrevocable copyright license to reproduce, prepare derivative works of, publicly display, publicly perform, sublicense, and distribute Your Contributions and such derivative works under the terms of the license specified  for this repository.
```

Comment:
Is this a license text used in existing MO codes?

Reply:
I've updated this, using the pattern applied within the Rose code base as a good practice guide